application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/MissingLicenseMessageMacro.xml

```diff
@@ -272,7 +284,7 @@
       <contentType>No content</contentType>
     </property>
     <property>
-      <defaultCategory/>
+      <defaultCategories/>
```

Comment:
I think we should hide this macro, since we only use it for pro apps. Having it in the "Internal" or "Deprecated" category will hide it.

https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Configuration/#HMacroscategoriesandvisibility

Reply:
I didn't knew about the existing categories. Adding it, thanks!